openshift_images/samples-operator-alt-registry.adoc

```diff
@@ -15,6 +15,11 @@ You must have access to the internet to obtain the necessary container images. I
 
 include::modules/installation-about-mirror-registry.adoc[leveloffset=+1]
 
+[NOTE]
```

Comment:
Same here - this note is now showing up twice back-to-back

Reply:
Addressed